public/app/dashboard/profile.html

```diff
@@ -1,4 +1,35 @@
 <div class="profile-container">
+  <div class="other-users-container">
+    <div>
+      <label for="findUser" class="sr-only">Search for users</label>
+      <oi-select id="find-user"
+        class="cleanMode"
+        name="findUser"
+        autofocus
+        oi-options="user.id as user.canvas_full_name for user in browse.otherUsers track by user.id | orderBy: canvas_full_name.toLowerCase()"
+        oi-select-options="{
+          dropdownFilter: 'userProfileSearchDropdown',
+          maxlength: 70
+        }"
+        ng-model="browse.nextUserId"
```

Comment:
@johncrossman Would it be worth renaming this to something like `browse.searchedUserId` to avoid confusion with `browse.next`?